driving_log_replayer_v2/driving_log_replayer_v2/launch/ground_segmentation.py

```diff
@@ -20,6 +20,7 @@
 """
 
 AUTOWARE_DISABLE = {
+    "sensing": "false",
```

Comment:
@badai-nguyen 
I think `sensing:false` is necessary based on the launch argument section in the documentation.